mobile/lib/constants/config.dart

```diff
@@ -65,12 +65,10 @@ class Config {
   static String get locationErrorMessage =>
       'Cannot get your location at the moment';
 
-  static int get maxSearchRadius => searchRadius * 2;
-
   static String get playStoreUrl =>
       'https://play.google.com/store/apps/details?id=com.airqo.app';
 
-  static int get searchRadius => 2;
+  static int get searchRadius => 4;
```

Comment:
Search radius in kilometers @Baalmart

Reply:
Thanks @NoahNsimbe . 4 KM is good. 
If something is beyond 4 KM, then it is definitely not nearby.